tests/logs/copy_repo/test_copy_repo/test_copy_container_repo.jsonl

```diff
@@ -0,0 +1,2 @@
+{"event": {"type": "check-repos-start"}}
```

Comment:
Why does your pull request show the introduction of all these `test_copy_repo` files?

You've probably pulled in some content from a PR that Nathan made while you were working on this. This is not ideal as the history will show both of you introducing these files, while they were really introduced by him.

Could you please rebase this PR and squash it into a single commit on top of the latest master?  That should make these go away.